test/FacebookAdsTest/Object/CustomAudienceNormalizers/EmailNormalizerTest.php

```diff
@@ -0,0 +1,59 @@
+
```

Comment:
this new empty line at the start of file breaks code if using HHVM as the PHP interpreter.
no more nits, promise! :-D